shared/js/angularProjects.js

```diff
@@ -1,82 +1,83 @@
-  'use strict';
+/* global $, angular */
```

Comment:
Quite possibly. I haven't actually touched the angular stuff yet. I think the downside to this is that dashboard probably already includes jquery, and this would result in a second copy of it in our bundle.